config/app.php

```diff
@@ -13,7 +13,7 @@
     |
     */
 
-    'name' => env('APP_NAME', 'Laravel'),
+    'name' => env('APP_NAME', 'blog'),
```

Comment:
Debería ser Blog API
```suggestion
    'name' => env('APP_NAME', 'blog API'),
```